src/compare-page-pull-request/compare-page-pull-request.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+import { h } from 'dom-chef';
+import elementReady from 'element-ready';
+import { getRepoURL } from '../page-detect';
+import 'selector-observer';
+
+const Constants = {
+    COMPARE_DETAILS_SELECTOR: '#branch-compare.content-container'
+};
```

Comment:
Kind of a nitpick, but if there's only one constant having it inside an object seems a little redundant. I would just have declared `const COMPARE_DETAILS_SELECTOR = '#branch-compare.content-container'`

Reply:
fair enough will do